docs/content/topics/scripting/workers.md

```diff
@@ -9,216 +9,117 @@ aliases:
+for processing an event.
+
+[Dependencies]: /topics/scripting/dependencies
+
+## Event Data
+
+Workers get the data they need to process an event via JSON mounted to its pod
+by Brigade, each time it runs. This JSON contains configuration that the worker
+itself uses, as well as project- and event-specific details. This file is
+located at `/var/event/event.json` inside the worker container.
+
+Note that custom workers may selectively disregard any data they deem
+inapplicable to the custom behavior they implement. For instance, a field that
+conveys the expected location of the `brigade.js` file is not applicable to a
+worker that does not utilize such configuration.
+
+### Fields
+
+Here's a rundown on the data present in the event JSON:
+
+| Field Name | Description |
+|------------|-------------|
+| `id` | A unique ID for the event |
+| `project.id` | The project ID that this event is associated with |
+| `project.kubernetes.namespace` | The Kubernetes namespace that the project uses |
+| `project.secrets` | The key/value map of [secrets] for the project |
+| `source` | The event [source] |
+| `type` | The event [type] |
+| `qualifiers` | The event [qualifiers] |
+| `labels` | The event [labels] |
+| `shortTitle` | A [short title] for the event |
+| `longTitle` | A [long title] for the event |
+| `payload` | The event [payload] |
+| `worker.apiAddress` | The address for the Brigade API server |
+| `worker.apiToken` | The token used for communicating with the Brigade API server |
+| `worker.logLevel` | The log level for worker logs |
+| `worker.configFilesDirectory` | The directory in the worker where the Brigade config files may be found |
+| `worker.defaultConfigFiles` | The default config files to use if not provided elsewhere (e.g. via a git repository's source code) |
+| `worker.git.cloneURL` | The clone URL of the git repository associated with the worker |
+| `worker.git.commit` | The commit SHA for the git repository |
+| `worker.git.ref` | The reference for the git repository |
+| `worker.git.initSubmodules` | Whether or not git submodules should be initialized |
+
+[secrets]: /topics/project-developers/secrets
+[source]: /topics/project-developers/events#source
+[type]: /topics/project-developers/events#type
+[qualifiers]: /topics/project-developers/events#qualifiers
+[labels]: /topics/project-developers/events#labels
+[short title]: /topics/project-developers/events#short-title
+[long title]: /topics/project-developers/events#long-title
+[payload]: /topics/project-developers/events#payload
 
 ## Job Pod Names and Labels
 
-To be visible to the rest of Brigade (and related projects, such as Kashti) and
-recognizable as pods that implement a build's job(s), certain naming and
-labeling conventions must be adhered to when a worker spawns such pods.
+To be visible to the rest of Brigade, certain naming and labeling conventions
+must be adhered to for job pods spawned by the worker.
```

Comment:
I think this whole section can go. It was applicable to v1 where workers created job pods directly. Now workers do this via API calls-- a big improvement because it means workers can't accidentally produce job pods that the rest of Brigade won't recognize as jobs.